scientific_library/contrib/tvb/contrib/scripts/datatypes/sensors.py

```diff
@@ -0,0 +1,172 @@
+# coding=utf-8
+from enum import Enum
+
+import numpy as np
+from tvb.contrib.scripts.datatypes.base import BaseModel
+from tvb.contrib.scripts.utils.data_structures_utils import labels_to_inds, monopolar_to_bipolar, \
+    split_string_text_numbers
+from tvb.basic.logger.builder import get_logger
+from tvb.basic.neotraits.api import Attr, NArray
+from tvb.datatypes.sensors import EEG_POLYMORPHIC_IDENTITY, MEG_POLYMORPHIC_IDENTITY, \
+    INTERNAL_POLYMORPHIC_IDENTITY
+from tvb.datatypes.sensors import Sensors as TVBSensors
+from tvb.datatypes.sensors import SensorsEEG as TVBSensorsEEG
+from tvb.datatypes.sensors import SensorsInternal as TVBSensorsInternal
+from tvb.datatypes.sensors import SensorsMEG as TVBSensorsMEG
+from tvb.simulator.plot.utils import ensure_list
+
+
+class SensorTypes(Enum):
```

Comment:
We should consider in the future such ENUMS as part of tvb-libray, next to our base Datatypes.
They will be generically useful

Reply:
I added a TODO here. When we will move this enum to the library we will have to review all usages of the sensor types harcoded strings in library.